examples/examples/http_proxy_middleware.rs

```diff
@@ -63,17 +66,17 @@ async fn main() -> anyhow::Result<()> {
 	println!("[main]: response: {:?}", response);
 
 	// Use hyper client to manually submit a `GET /health` request.
-	let http_client = Client::new();
+	let http_client = Client::builder(TokioExecutor::new()).build_http();
```

Comment:
Interesting way to be agnostic over runtime; I could imagine we might adopt a similar thing in subxt to hide the spawning stuff behind the scenes unless the user wants to override the executor or manually drive stuff